app/models/stats/validator_score.rb

```diff
@@ -55,7 +56,9 @@ def root_distance_stats(with_history: false)
     end
 
     def vote_distance_all_history
-      @vote_distance_all_history ||= relation&.joins(:validator)&.pluck(:vote_distance_history, :account)
+      @vote_distance_all_history ||= relation&.joins(:validator)
+                                             &.where("validators.is_active = ? AND validator_score_v1s.delinquent = ?", true, false)
+                                             &.pluck(:vote_distance_history, :account)
```

Comment:
Does the rescue method &. is required? Or maybe we can get rid of that.

```suggestion
      @vote_distance_all_history ||= 
        relation.joins(:validator)
                     .where("validators.is_active = ? AND validator_score_v1s.delinquent = ?", true, false)
                     .pluck(:vote_distance_history, :account)
```